### PR TITLE
Create and Remove books

### PR DESCRIPTION
**Pull request by Somdotta07**
- Added book's action and reducer using _ducks pattern_
- Configured the Redux store
-  Implemented React component by dispatching a corresponding action
- Used the redux-logger

### CODE REVIEW

File: src/redux/books/book.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { createSlice, current } from '@reduxjs/toolkit';
+
+const bookSlice = createSlice(
```
Comment (er-contreras) [inline src/redux/books/book.js:3]: Good job using createSlice from the redux toolkit 👍🏼

File: src/redux/configureStore.js [hunk 2]
```diff
@@ -1,8 +1,10 @@
 import { configureStore } from '@reduxjs/toolkit';
+import logger from 'redux-logger';
+import booksReducer from './books/book';
 
 const store = configureStore({
-  reducer: {
-  },
+  reducer: { books: booksReducer },
+  middleware: (getDefaultMiddleware) => getDefaultMiddleware().concat(logger),
```
Comment (er-contreras) [inline src/redux/configureStore.js:7]: - Good job setting the configure store 😃

**Review by er-contreras — APPROVED** [review]
Hi @Somdotta07, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- Well done!

Felix

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/book.js (hunk 1 above)
Comment (Somdotta07) [reply src/redux/books/book.js:3]: Thank you

File: src/redux/configureStore.js (hunk 2 above)
Comment (Somdotta07) [reply src/redux/configureStore.js:7]: Thank you

**Pull request merged**